docarray/store/helpers.py

```diff
@@ -131,12 +132,27 @@ def close(self):
 
 
 def _to_binary_stream(
+    self, protocol: Literal['pickle', 'protobuf'],
```

Comment:
default value should still be `protobuf` here and we should not change the order of the argument (protocol should stay at the same place as before)

Reply:
I would want to make it clear if the `protocol` is in the right place and order (i.e. 2nd argument).

Is my understanding correct? At the same time let me make sure to add `protobuf ` as the default value.

Reply:
Also, if we want `protocol` to act as a default argument then it has to be placed at the end.

Reply:
I think you can do what you did on the other and kept it at the end